provider.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source                = "hashicorp/aws"
-      version               = "~> 4"
+      version               = "~> 5"
```

Comment:
Thanks for the PR. I've got same issue.
I think better constraint would be `">= 4"` to prevent conflict with other modules which use versions older than 5.

Reply:
Thanks for the offer! So I will do

Reply:
Thanks, I think it should be `">= 4"` not 5

Reply:
Thanks, I think it should be `">= 4"` not 5